scripts/package

```diff
@@ -5,21 +5,53 @@ source $(dirname $0)/version
 
 cd $(dirname $0)/..
 
-PROJECT=`basename "$PWD"`
+project=$(basename "$PWD")
 
-if [ ! -x ./bin/longhorn ]; then
-    ./scripts/build
+command -v buildx >/dev/null && build_cmd=(buildx) || build_cmd=(docker buildx)
+
+# read configurable parameters
+REPO=${REPO:-longhornio}
+IMAGE_NAME=${IMAGE_NAME:-$project}
+TAG=${TAG:-''}
+OUTPUT_ARGS=${OUTPUT_ARGS:-'--load'}
+IS_SECURE=${IS_SECURE:-'false'}
+MACHINE=${MACHINE:-''}
+TARGET_PLATFORMS=${TARGET_PLATFORMS:-''}
+IID_FILE=${IID_FILE:-''}
+IID_FILE_FLAG=${IID_FILE_FLAG:-''}
+
+if [[ -z $TAG ]]; then
```

Comment:
```suggestion
if [[ -z ${TAG} ]]; then
```
Suggest maintaining consistency and this improves readability. Same suggestion for the rest of the PR.